frontend/.angular-cli.json

```diff
@@ -20,6 +20,7 @@
       "prefix": "app",
       "styles": [
         "../node_modules/bootstrap/dist/css/bootstrap.min.css",
+        "assets/css/bootstrap-glyphicons.css",
```

Comment:
Maybe it's beside this pull request, but can we try to use something that install from npm (e.g. https://www.npmjs.com/package/font-awesome) instaed of pusting whole source code in our repository?